lib/spack/docs/basic_usage.rst

```diff
@@ -2,7 +2,7 @@
 
 Basic usage
 =====================
-
+p
```

Comment:
Yes.  I made the edit in the GitHub UI and must have hit the p key without noticing...

Perhaps cleanest is to just close the pull request and make another one?